Project.toml

```diff
@@ -30,6 +30,9 @@ ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
+[extras]
```

Comment:
Why is only Zygote added here?

Reply:
It doesn't matter for our tests because we have test/Project.toml. But Pkg requires it, otherwise it's not possible to add a compat entry for a weak dependency.

Reply:
Maybe we should just remove the test/Project.toml if possible (IIRC it helped with some dependency incompatibilities a while ago but maybe that's not a problem anymore).